core/templates/domain/improvements/ExplorationTaskObjectFactorySpec.ts

```diff
@@ -107,6 +107,6 @@ describe('Exploration task object factory', () => {
     expect(
       () => expTaskObjectFactory.createFromBackendDict(
         this.newTaskEntryBackendDict('unknown_task_type'))
-    ).toThrowError('unsupported task type: unknown_task_type');
+    ).toThrowError(/unsupported task type/);
```

Comment:
I suggest checking the whole thing, to make sure that the error message actually provides debuggable information.

Reply:
Done.